svf-llvm/include/SVF-LLVM/SVFIRBuilder.h

```diff
@@ -235,9 +232,15 @@ class SVFIRBuilder: public llvm::InstVisitor<SVFIRBuilder>
 
     /// Handle external call
     //@{
-    virtual void parseOperations(std::vector<ExtAPI::Operation>  &operations, const SVFCallInst* svfcall);
+    virtual SVFCallInst* getCreatedExtFunCallInst(const std::string funName, ExtAPI::ExtFunctionOps &extFunctionOps);
```

Comment:
Can you move the implementation/body of all external call-related APIs to a separate cpp file, for example, SVFIRExtAPI.cpp? You can keep their declaration here in SVFIRBuilder.h.

Reply:
Ok